topology/University of Chicago/UChicago/UChicago_OSGConnect.yaml

```diff
@@ -13,63 +13,330 @@ Resources:
     ContactLists:
       Administrative Contact:
         Primary:
+          Name: Judith Stephen
```

Comment:
Can you also make a PR for flocking-hosts.yml in osg-flock to make Judith the admin for the hosts she's going to be the primary contact for?

Reply:
Hrm, I thought I had left a comment: @rynge will be working on ripping out `flocking-hosts.yaml` in favor of what's in Topology so I don't think it's necessary.